src/store/RootStore/hooks/useCartStoreInit.ts

```diff
@@ -0,0 +1,6 @@
+import rootStore from "../instance";
+
+export const useCartStoreInit = (): void => {
+  const cartItems = JSON.parse(localStorage.getItem("cart") || "");
```

Comment:
так делать опасно:
![image](https://user-images.githubusercontent.com/71431554/225515235-bf9f7c01-09af-4f33-b460-5edeeb0f1b2f.png)

лучше что-то типа
```suggestion
  const cartField = localStorage.getItem("cart");
  const cartItems = cartField ? JSON.parse(cartField) : [];
```